### PR TITLE
dnsdist: More words on TLS session encryption keys setup and rotation

### PR DESCRIPTION
**Pull request by rgacogne**
### Short description

Related to https://github.com/PowerDNS/pdns/issues/13336. 

### Checklist

I have:
- [x] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled this code
- [ ] tested this code
- [x] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)

### CODE REVIEW

File: pdns/dnsdistdist/docs/advanced/tuning.rst [hunk 1]
```diff
@@ -116,6 +116,10 @@ Incoming and outgoing DNS over TLS, as well as outgoing DNS over HTTPS, might be
  * supported ciphers depend on the exact kernel version used. ``TLS_AES_128_GCM_SHA256`` might be a good option for testing purpose since it was supported pretty early
  * as of OpenSSL 3.0.7, kTLS can only be used for sending TLS 1.3 packets, not receiving them. Both sending and receiving packets should be working for TLS 1.2.
 
+TLS performance
+---------------
+
+For DNS over HTTPS and DNS over TLS, in addition to the advices above we advise reading the :doc:`tls-sessions-management` page to know how to improve TLS session resumption ratio, which has a huge impact on CPU usage and latency.
```
Comment (Habbie) [inline pdns/dnsdistdist/docs/advanced/tuning.rst:122]:
```suggestion
For DNS over HTTPS and DNS over TLS, in addition to the advice above we suggest reading the :doc:`tls-sessions-management` page to learn how to improve TLS session resumption ratio, which has a huge impact on CPU usage and latency.
```

**Review by Habbie — CHANGES_REQUESTED** [review]
wonderful explanations. Couple of nits

**Comment by rgacogne** [comment]
> Couple of nits

Thanks! I have now pushed a new commit fixing them.

**Comment by rgacogne** [comment]
Rebased on master to fix a spell-checking issue.

**Review by Habbie — APPROVED** [review]

**Pull request merged**